controls/pcidss_ocp4.yml

```diff
@@ -144,68 +181,95 @@ controls:
       that provide authorized publicly accessible services, protocols, and ports.
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    notes: |-
+      To a limited degree, OpenShift's ingress routing (enabled by default)
+      and NetworkPolicies can support restriction of inbound traffic to pods.
+      Further restricting needs to happen on the cloud where OpenShift is deployed
+      at, but that's outside of OpenShift's scope.
+    status: automated
+    rules:
+      - configure_network_policies_namespaces
 
   - id: Req-1.3.2
     title: 1.3.2 Limit inbound Internet traffic to IP addresses within the DMZ.
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    notes: |-
+      To a limited degree, OpenShift's ingress routing (enabled by default)
+      and NetworkPolicies can support restriction of inbound traffic to pods.
+      Further restricting needs to happen on the cloud where OpenShift is deployed
+      at, but that's outside of OpenShift's scope.
+    status: automated
+    rules:
+      - configure_network_policies_namespaces
 
   - id: Req-1.3.3
-    title: 1.3.3 Do not allow any direct connections inbound or outbound for traffic
-      between the Internet and the cardholder data environment.
+    title: >-
+      1.3.3 Implement anti-spoofing measures to detect and block forged
+      source IP addresses from entering the network.
+    description: (For example, block traffic originating from the Internet with an internal
+      source address.)
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      Anti-spoofing and blocking forged IP addresses is outside the scope
+      of the OpenShift Container Platform.
+    status: not applicable
     rules: []
 
   - id: Req-1.3.4
-    title: 1.3.4 Implement anti-spoofing measures to detect and block forged source
-      IP addresses from entering the network.
-    description: (For example, block traffic originating from the Internet with an internal
-      source address.)
+    title: >-
+      1.3.4 Do not allow unauthorized outbound traffic from the cardholder
+      data environment to the Internet.
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      OpenShift can support authorization of outbound traffic via
+      egress  policy  objects[1][2].
+
+      [1] https://docs.openshift.com/container-platform/4.6/networking/openshift_sdn/configuring-egress-firewall.html
+      [2] https://docs.openshift.com/container-platform/4.6/networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.html
+    status: supported
```

Comment:
Should we have a CaC rule for this control?

Reply:
yes. We should; that'll come in a separate PR... at some point

Reply:
OK